app/src/main/java/com/globant/weatherapp/mvp/model/WeatherModel.kt

```diff
@@ -1,6 +1,37 @@
 package com.globant.weatherapp.mvp.model
 
 import com.globant.weatherapp.mvp.contracts.WeatherContracts
+import org.json.JSONArray
 
-class WeatherModel: WeatherContracts.Model {
+class WeatherModel(private val cities: String) : WeatherContracts.Model {
+
+    private var cityList: MutableList<String> = mutableListOf()
+    private var cityIdList: MutableList<Int> = mutableListOf()
```

Comment:
This can be " val something : List<Int> = mutableListOf()"

Reply:
Done!